app/src/main/res/layout/dashboard_page.xml

```diff
@@ -120,80 +120,201 @@
-                            android:layout_gravity="center_vertical"
-                            android:layout_marginTop="22dp">
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_weight="1"
+                            android:orientation="vertical">
+
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:text="0.98k"
+                                android:textSize="14sp"
+                                android:textStyle="bold"
+                                android:textColor="@color/black"/>
 
                             <TextView
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
-                                android:text="Year"
-                                android:textStyle="bold" />
+                                android:text="+0.07%"
+                                android:textSize="12sp"
+                                android:textColor="#4CAF50"/>
+
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:text="6.54k"
+                                android:textSize="14sp"
+                                android:textStyle="bold"
+                                android:textColor="@color/black"/>
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:text="+0.07%"
+                                android:textSize="12sp"
+                                android:textColor="#4CAF50"/>
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:text="70.98k"
+                                android:textSize="14sp"
+                                android:textStyle="bold"
+                                android:textColor="@color/black"/>
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:text="+0.07%"
+                                android:textSize="12sp"
+                                android:textColor="#4CAF50"/>
+
+
 
                         </LinearLayout>
+
                     </LinearLayout>
 
                 </LinearLayout>
             </androidx.cardview.widget.CardView>
 
+
+
```

Comment:
Use GridLayout for efficient layouting, no need for too much nesting of linear layouts.
Also, try to center the GridLayout at the center of card, you can try using gravity:center.

Check this: https://techvidvan.com/tutorials/android-gridlayout/